src/PhpSpreadsheet/Reader/Xml.php

```diff
@@ -617,9 +617,11 @@ public function loadIntoExisting($pFilename, Spreadsheet $spreadsheet)
                     ++$rowID;
                 }
 
-                $xmlX = $worksheet->children($namespaces['x']);
-                if (isset($xmlX->WorksheetOptions)) {
-                    (new PageSettings($xmlX, $namespaces))->loadPageSettings($spreadsheet);
+                if (isset($namespaces['x'])) {
```

Comment:
Please ignore comments here. I'm just experimenting with the PR review features in PHPStorm